src/usr/local/www/system_certmanager.php

```diff
@@ -93,9 +95,13 @@
 
 if ($act == "new") {
 	$pconfig['method'] = $_POST['method'];
+	$pconfig['keytype'] = "RSA";
 	$pconfig['keylen'] = "2048";
+	$pconfig['eckey'] = "brainpoolP160r1";
```

Comment:
See previous comment about the default curve.

Reply:
fixed